components/brave_wallet/browser/simulation_responses.idl

```diff
@@ -0,0 +1,270 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Dummy comments to make the linter happy.
+namespace simulation_responses {
+  dictionary SolanaPrice {
+    DOMString source;
+
+    // FIXME: The fields below should be in camelCase. Needs an upstream fix.
```

Comment:
what was the issue you encountered, can you elaborate?

Reply:
`simulation_responses::SolanaPrice::FromValue(value.GetDict())` returns an `absl::nullopt` if the `.idl` definition is `lastUpdatedAt` (camelCase) but the actual response is `last_updated_at` (snake_case).

I've communicated this to the Blowfish team and they will be fixing it soon.

P.S. There was a missing check for `absl::nullopt` as return value of `simulation_responses::SolanaPrice::FromValue()` which would've resulted in a crash. It's now being handled correctly ➜ 59514d0119cffccb004ba1c54596ac1047b21f9f.